api/osrd_infra/models/rolling_stock.py

```diff
@@ -99,3 +99,43 @@ def import_railjson(rolling_stock_dict: Dict, force: bool = False):
         if force:
             return RollingStock.objects.update_or_create(**rs_obj.dict())
         return RollingStock.objects.create(**rs_obj.dict())
+
+
+class RollingStockImage(models.Model):
+    image = models.BinaryField(null=False, blank=False)
+
+    class Meta:
+        abstract = True
+        verbose_name_plural = "rolling stock images"
+
+
+class RollingStockCompoundImage(RollingStockImage):
+    class Meta:
+        db_table = "osrd_infra_rollingstockimage"
+        managed = False
+        verbose_name_plural = "rolling stock compound images"
+
+
+class RollingStockLivery(models.Model):
+    name = models.CharField(max_length=255, help_text=_("Name of the livery"), default="default")
+    rolling_stock = models.ForeignKey(RollingStock, related_name='liveries', on_delete=models.CASCADE)
+    # OneToOne relationship
+    compound_image = models.OneToOneField(RollingStockCompoundImage, null=True, on_delete=models.CASCADE)
```

Comment:
```suggestion
    compound_image = models.OneToOneField(RollingStockCompoundImage, null=True, on_delete=models.CASCADE)
```
Comment is paraphrasing the code, could be removed I think